### PR TITLE
Rename specular-glossiness parameters to their canonical names.

### PR DESCRIPTION
**Pull request by prideout**

### CODE REVIEW

File: docs/Materials.md.html [hunk 1]
```diff
@@ -527,25 +527,32 @@
 
 This alternative lighting model exists to comply with legacy standards.
 
-Similar to cloth, it encompasses all the parameters previously defined for the standard
-lit mode except for `metallic` and `reflectance`, and adds a `sheenColor` parameter. The sheen
-color to directly control the _specular color_ of the material.
+This encompasses all the parameters previously defined for the standard lit mode except for
+_metallic_ and _reflectance_, and adds parameters for _specularColor_ and _glossiness_. The specular
```
Comment (romainguy) [inline docs/Materials.md.html]: And `roughness` is removed too.

File: docs/Materials.md.html [hunk 2]
```diff
@@ -527,25 +527,32 @@
...
+color is used to formulate the BRDF inputs as follows.
+
+```glsl
+pixel.diffuseColor = baseColor * (1.0 - max(specularColor.r, specularColor.g, specularColor.b));
```
Comment (romainguy) [inline docs/Materials.md.html]: Don't put the math/code in this document, this is meant for users. Just explain that base color is diffuse and specular color is the specular color.

File: libs/filamat/src/shaders/CodeGenerator.cpp [hunk 3]
```diff
@@ -509,6 +509,8 @@ char const* CodeGenerator::getConstantName(MaterialBuilder::Property property) n
         case Property::SUBSURFACE_POWER:     return "SUBSURFACE_POWER";
         case Property::SUBSURFACE_COLOR:     return "SUBSURFACE_COLOR";
         case Property::SHEEN_COLOR:          return "SHEEN_COLOR";
+        case Property::GLOSSINESS:          return "GLOSSINESS";
```
Comment (romainguy) [inline libs/filamat/src/shaders/CodeGenerator.cpp]: alignment of "return" on both lines. Tabs?

File: shaders/src/common_material.fs [hunk 4]
```diff
@@ -81,13 +81,18 @@ struct MaterialInputs {
...
-#if defined(SHADING_MODEL_CLOTH) || defined(SHADING_MODEL_SPECULAR_GLOSSINESS)
+#if defined(SHADING_MODEL_CLOTH)
     vec3  sheenColor;
 #if defined(MATERIAL_HAS_SUBSURFACE_COLOR)
     vec3  subsurfaceColor;
 #endif
 #endif
 
+#if defined(SHADING_MODEL_SPECULAR_GLOSSINESS)
+    vec3  specularColor;
```
Comment (romainguy) [inline shaders/src/common_material.fs]: You need to remove `roughness` when the model is specular glossiness

File: docs/Materials.md.html (hunk 1 above)
Comment (prideout) [reply docs/Materials.md.html]: Done

File: docs/Materials.md.html (hunk 2 above)
Comment (prideout) [reply docs/Materials.md.html]: Removed

File: libs/filamat/src/shaders/CodeGenerator.cpp (hunk 3 above)
Comment (prideout) [reply libs/filamat/src/shaders/CodeGenerator.cpp]: Fixed

File: shaders/src/common_material.fs (hunk 4 above)
Comment (prideout) [reply shaders/src/common_material.fs]: Done

**Pull request merged**